content/v2.0/monitor-alert/templates/AWS.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
- Filenames should be lowercase: `content/v2.0/monitor-alert/templates/aws.md`
- I don't know if `templates` is the right directory name for this collection of guides. Templates are the quickest way to setup monitoring solutions, but they aren't the only way. While the first wave of these docs will all use templates to get set up, I don't want to pigeon-hole this guides into only using templates to setup monitoring. What about 'solutions', 'guides', 'services'? Other thoughts?

Reply:
I like "solutions" as a general term for all the ways folks can monitor. Templates could be a subset of solutions.

Reply:
File name still needs to be lowercased. This matters to our production webserver.

Reply:
Thanks, Scott, interesting...I updating filename in VSCode doesn't appear to work. Updated directly in github.